src/components/Link.tsx

```diff
@@ -260,7 +263,6 @@ export function InlineLink({
       style={[
         {color: t.palette.primary_500},
         (hovered || focused || pressed) && {
-          outline: 0,
```

Comment:
Yeah leave this here, since it removes the ugly focus outlines on web. We have an underline to serve as a focus state. We should replace with this to avoid errors `...web({ outline: 0 })`.